### PR TITLE
PIMS-1401,1402,1464 add RAEG/SPP to buildings.

### PR DESCRIPTION
**Pull request by devinleighsmith**
Also misc lint and test fixes.

### CODE REVIEW

File: frontend/src/actions/parcelsActions.tsx [hunk 1]
```diff
@@ -33,6 +33,7 @@ export interface IAddress {
 export interface IBuilding {
   id: number;
   localId: string;
+  projectNumber: string;
```
Comment (Fosol) [inline frontend/src/actions/parcelsActions.tsx:36]: It can be null

File: frontend/src/contexts/authStateContext.tsx [hunk 2]
```diff
@@ -1,5 +1,4 @@
 import * as React from 'react';
-import { useKeycloak } from '@react-keycloak/web';
```
Comment (Fosol) [inline frontend/src/contexts/authStateContext.tsx:2]: I was fairly certain when I removed this something broke.  I didn't understand why as it doesn't appear to be used in the file.

File: frontend/src/components/maps/BuildingPopupView.tsx [hunk 3]
```diff
@@ -48,6 +48,12 @@ export const BuildingPopupView: React.FC<IBuildingDetailProps> = ({ building })
                 <Label>Predominate Use: </Label>
                 {building?.buildingPredominateUse}
               </ListGroup.Item>
+              {building.projectNumber && (
```
Comment (asanchezr) [inline frontend/src/components/maps/BuildingPopupView.tsx:51]: Maybe `building?. projectNumber` ? To match the surrounding code?

File: frontend/src/actions/parcelsActions.tsx [hunk 4]
```diff
@@ -70,6 +71,7 @@ export interface IParcel {
   id: number | any;
   pid: string | undefined;
   pin: number | any;
+  projectNumber: string;
```
Comment (Fosol) [inline frontend/src/actions/parcelsActions.tsx:74]: Does this one need to be null too?

Comment (devinleighsmith) [reply frontend/src/actions/parcelsActions.tsx:74]: will add that to my fiscal changes.

**Pull request merged**